tests/numeric_comparison_tests.py

```diff
@@ -58,3 +58,17 @@ def less_than_or_equal_to_matches_when_actual_is_less_than_or_equal_to_value():
 def less_than_or_equal_to_description_describes_value():
     matcher = less_than_or_equal_to(42)
     assert_equal("less than or equal to 42", matcher.describe())
+
+
+@istest
+def close_to_matches_when_actual_is_close_to_value_plus_delta():
+    matcher = close_to(42, 1)
+    assert_equal(matched(), matcher.match(43))
+    assert_equal(matched(), matcher.match(41))
```

Comment:
I think these two positive cases only test the boundaries. Could we also test a value not on the boundary?